crates/swc_css_ast/src/at_rule.rs

```diff
@@ -801,3 +804,36 @@ pub enum SizeFeatureName {
     #[tag("Ident")]
     Ident(Ident),
 }
+
+#[ast_node("ExtensionName")]
+#[derive(Eq, Hash)]
+pub struct ExtensionName {
+    pub span: Span,
+    #[cfg_attr(feature = "rkyv", with(swc_atoms::EncodeJsWord))]
+    pub value: JsWord,
+    #[cfg_attr(feature = "rkyv", with(swc_atoms::EncodeJsWord))]
+    pub raw: Option<JsWord>,
```

Comment:
Let's use `Option<Atom>` instead

Reply:
@kdy1 Let's do it in another PR, we need to use it in more places, so I want to do it in one PR. What do you think?